lib/src/ui/screens/checkup/checkup.dart

```diff
@@ -1,5 +1,6 @@
 import 'package:covidnearme/src/blocs/questions/questions.dart';
 import 'package:covidnearme/src/data/repositories/questions.dart';
+import 'package:covidnearme/src/ui/screens/thankyou/thankyou.dart';
```

Comment:
Non-blocking: you could instead import the router here.
```suggestion
import 'package:covidnearme/src/ui/router.dart';
```